app/src/main/res/values/settings_keys.xml

```diff
@@ -1031,12 +1036,15 @@
         <item>@string/systems_language</item>
         <item>Basa Acèh</item>
         <item>العربية</item>
+        <item>العربية (ليبيا)</item>
```

Comment:
Why the country name here and not just the code? And why the () before but for the languages a few lines further after?

Reply:
I think it is good. No country code, because this would be in Latin letters and we'd have to deal with RTL and LTR in one line. And on the left, because that's RTL, so still at the end of the line

Reply:
Oh, I incorrectly assumed Bengali was RTL as well, but apparently it's just LTR. My other point still stands, i.e. changing e.g. "Nederlands (BE)" to "Nederlands (België)" for more consistency.

Reply:
Oh. I didn't see you reply. I'd say we can put that on our list for the next release, but not this one.